config/module.config.php

```diff
@@ -43,8 +43,8 @@
     ],
     'validators' => [
         'factories' => [
-            Validator\DbRecordExists::class => Validator\Db\RecordExistsFactory::class,
-            Validator\DbNoRecordExists::class => Validator\Db\NoRecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbRecordExists' => Validator\Db\RecordExistsFactory::class,
+            'ZF\ContentValidation\Validator\DbNoRecordExists' => Validator\Db\NoRecordExistsFactory::class,
```

Comment:
Thanks for you feedback, we noticed that! you can check this thread : https://github.com/zfcampus/zf-content-validation/pull/76#discussion_r71698040